pkgs/development/python-modules/openmm/default.nix

```diff
@@ -0,0 +1,87 @@
+{ buildPythonPackage, lib, fetchFromGitHub, cmake, clang, gfortran, fftwSinglePrec
+, doxygen, ocl-icd, opencl-headers, swig, python, cython, numpy
+, cudatoolkit, enableCuda ? false
+} :
+
+buildPythonPackage rec {
+  pname = "openmm";
+  version = "7.7.0";
+
+  src = fetchFromGitHub {
+    owner = "openmm";
+    repo = pname;
+    rev = version;
+    hash = "sha256-2PYUGTMVQ5qVDeeABrwR45U3JIgo2xMXKlD6da7y3Dw=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    gfortran
+    swig
+    cython
+    doxygen
+  ];
+
+  buildInputs = [
+    fftwSinglePrec
+    ocl-icd
+    opencl-headers
+  ] ++ lib.optional enableCuda cudatoolkit
+  ;
+
+  propagatedBuildInputs = [
+    python
```

Comment:
Maybe it would be better to explicitly specify that this is `python2`?

Reply:
is gone